package.json

```diff
@@ -30,14 +30,13 @@
     "jasmine-ajax": "^4.0.0",
     "jasmine-jquery": "^2.1.1",
     "leaflet": "^1.4.0",
+    "leaflet-blurred-location": "git://github.com/publiclab/leaflet-blurred-location.git#main",
```

Comment:
Shall we pin this to a version instead of to the github directly? What do you think?

Reply:
Sure! Honestly I was surprised that it wasn't using a version number.